v2/internal/reconcilers/reconcile_policy.go

```diff
@@ -11,29 +11,24 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/Azure/azure-service-operator/v2/internal/util/to"
+	"github.com/Azure/azure-service-operator/v2/pkg/common"
 )
 
-// ReconcilePolicyAnnotation describes the reconcile policy for the resource in question.
-// A reconcile policy describes what action (if any) the operator is allowed to take when
-// reconciling the resource.
-// If no reconcile policy is specified, the default is "run"
-const ReconcilePolicyAnnotation = "serviceoperator.azure.com/reconcile-policy"
-
 type ReconcilePolicy string
 
 const (
 	// ReconcilePolicyManage instructs the operator to manage the resource in question.
-	// This includes issuing PUTs to update it and DELETE's to delete it from Azure if deleted in Kuberentes.
+	// This includes issuing PUTs to update it and DELETE's to delete it from Azure if deleted in Kubernetes.
 	// This is the default policy when no policy is specified.
-	ReconcilePolicyManage = ReconcilePolicy("manage")
+	ReconcilePolicyManage = ReconcilePolicy(common.ReconcilePolicyManage)
 
 	// ReconcilePolicySkip instructs the operator to skip all reconciliation actions. This includes creating
 	// the resource.
-	ReconcilePolicySkip = ReconcilePolicy("skip")
+	ReconcilePolicySkip = ReconcilePolicy(common.ReconcilePolicySkip)
 
 	// ReconcilePolicyDetachOnDelete instructs the operator to skip deletion of resources in Azure. This allows
 	// deletion of the resource in Kubernetes to go through but does not delete the underlying Azure resource.
-	ReconcilePolicyDetachOnDelete = ReconcilePolicy("detach-on-delete")
+	ReconcilePolicyDetachOnDelete = ReconcilePolicy(common.ReconcilePolicyDetachOnDelete)
 )
 
```

Comment:
I suggest moving all of this into `common` - and get rid of the string typed constants. Consumers can then use the strongly typed `ReconcilePolicyManage` and cast it to a string if they must.

Reply:
This also gets rid of the need to document them in two places.

Reply:
Agree with @theunrepentantgeek